step.yml

```
@@ -64,18 +64,37 @@ toolkit:
     package_name: github.com/bitrise-steplib/steps-ios-auto-provision-appstoreconnect
 
 inputs:
-  - build_api_token: $BITRISE_BUILD_API_TOKEN
+  - connection: "automatic"
     opts:
-      title: Build API token
+      title: Bitrise Apple Developer Connection
+      summary: The Apple Service authentication method the Step uses. By default, Bitrise Apple Developer connection based on API key is used and other authentication-related Step inputs are ignored.
       description: |-
-        Every build gets a temporary Bitrise API token to download the connected API key in a JSON file.
+        The input determines the method used for Apple Service authentication. By default, Bitrise Apple Developer connection based on API key is used and other authentication-related Step inputs are ignored.
+        
+        You can choose if Bitrise Apple Developer connection to use or you can tell the Step to only use Step inputs for authentication:
+        - `automatic`: Use Apple Developer connection based on API key. Step inputs are only used as a fallback.
+        - `api_key`: Use the Apple Developer connection based on API key authentication. Authentication-related Step inputs are ignored.
+        - `off`: Do not use any Apple Developer Connection. Only authentication-related Step inputs are considered.
       is_required: true
-  - build_url: $BITRISE_BUILD_URL
+      value_options:
+      - "automatic"
+      - "api_key"
+      - "off"
+  - api_key_path: ""
     opts:
-      title: Build URL
+      title: "API Key URL"
+      summary: Path to local or remote file
       description: |-
-        URL of the current build or local path URL to your apple_developer_portal_data.json.
-      is_required: true
+        Specify the path in an URL format where your API key is stored.
+        For example: `https://URL/TO/AuthKey_something.p8` or `file:///PATH/TO/AuthKey_something.p8`
+        You can upload your key on the **Generic File Storage** tab in the Workflow Editor and set the environment variable for the file here.
+        For example: `$BITRISEIO_MYKEY_URL`
```

```suggestion
        Specify the path in an URL format where your API key is stored.
        
        For example: `https://URL/TO/AuthKey_something.p8` or `file:///PATH/TO/AuthKey_something.p8`
        
        You can upload your key on the **Generic File Storage** tab in the Workflow Editor and set the environment variable for the file here.
        
        For example: `$BITRISEIO_MYKEY_URL`
```
Adding newlines for better readability.